video/out/gpu/libmpv_gpu.c

```diff
@@ -111,6 +111,7 @@ static int set_parameter(struct render_backend *ctx, mpv_render_param param)
         return 0;
     }
     case MPV_RENDER_PARAM_AMBIENT_LIGHT: {
+        MP_WARN(ctx, "MPV_RENDER_PARAM_AMBIENT_LIGHT is deprecated and might be removed in the future (no replacement)\n");
```

Comment:
It seems to be used by this guy https://github.com/mpv-player/mpv/blob/7899a28699e45ad407f22102e7fe5ba34228173f/video/out/cocoa_cb_common.swift#L113-L115 now it will write warning on every change on ambient light? Maybe we should keep it as macOS only feature, but not deprecate in fact? I'm not sure what's best.

Reply:
you are right, i didn't think about the log spamming.

since i am planning to add an alternative approach for doing the same thing, it probably makes sense to leave the deprecation commit of the libmpv API out of this. when i add the alternative approach and removed the API usage from cocoa-cb we can also properly deprecated the libmpv API?